sycl/test/basic_tests/handler/handler_generic_lambda_interface.cpp

```diff
@@ -52,5 +53,21 @@ int main() {
   test_parallel_for_work_group<class Group2Name, sycl::group<2>>(sycl::range<2>{1, 1});
   test_parallel_for_work_group<class Group3Name, sycl::group<3>>(sycl::range<3>{1, 1, 1});
 
+#ifndef SYCL2020_CONFORMANT_APIS
+  // For backwards compatibility, we still support some invalid variants of
+  // kernel lambdas. These tesst cases should be removed once we make conformant
```

Comment:
```suggestion
  // kernel lambdas. These test cases should be removed once we make conformant
```

Reply:
Applied in f3edff180367d853aa2c2b8cad90aa1e5bf3690b